.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: xenial
+sudo: required
+
+language: generic
+
+addons:
+  apt:
+    packages:
+     - build-essential
+     - make
+     - perl
+     - git
+     - unzip
+     - m4
+     - gcc
+     - curl
+     - libpcre3
+     - libpcre3-dev
+     - zlib1g-dev
+
+jobs:
+  include:
+    # Known builds we want to succeed
+    - script: ./kong-ngx-build --prefix prefix --openresty 1.11.2.5 --openssl 1.0.2n --luarocks 2.4.3 --no-openresty-patches
```

Comment:
To verify that the `--no-openresty-patches` command succeeded, we should ensure that `work/openresty-1.11.2.5/bundle/.patch_applied` does not exist.

Reply:
I was seeing these more like green path integration tests. IIRC the no-openresty-patches flag was needed for builds with openssl versions 1.0.x to succeed.

Reply:
The flag was added to build OpenResty without the patches. We don’t apply any patch to OpenSSL ourselves. There are no patches for OpenSSL 1.11.2.5, but I would expect future maintainers of this test harness to duplicate the test jobs and increase the matrix size. Among these jobs will be OpenResty 1.13.6.* builds with and without patches. All builds without patches should perform explicit checks that those were not applied imho, regardless of their version (since the version number comes from the test matrix). Again, not a blocker so we can take care of this later.

Reply:
Still looking at this one